### PR TITLE
block  io.open

### PR DESCRIPTION
**Pull request by mohitkhullar**
- bug fix

### CODE REVIEW

**Comment by akshatsikarwar** [comment]
Didn't we want to stub out entire `io` module?

**Comment by mohitkhullar** [comment]
blocked the write for now, read mode can be used to read csv files.

File: lua/liolib.c [hunk 1]
```diff
@@ -161,6 +161,9 @@ static int io_tostring (lua_State *L) {
 static int io_open (lua_State *L) {
   const char *filename = luaL_checkstring(L, 1);
   const char *mode = luaL_optstring(L, 2, "r");
+  if (strcmp(mode, "r") != 0) {
+    return luaL_error(L, "Only read mode allowed");
+  }
```
Comment (nirbhayc) [inline lua/liolib.c]: IMHO, even OP user should not be allowed to read from arbitrary location. We could override this default behavior via a tunable.

**Comment by nirbhayc** [comment]
Also consider providing a way to enable it back, in case this feature is being used in prod.

**Pull request merged**